core/templates/dev/head/components/OutcomeDestinationEditorDirective.js

```diff
@@ -57,8 +55,11 @@ oppia.directive('outcomeDestinationEditor', [
           // We restrict editing of refresher exploration IDs to
           // admins/moderators for now, since the feature is still in
           // development.
-          $scope.canEditRefresherExplorationId = (
-            GLOBALS.isAdmin || GLOBALS.isModerator);
+          UserService.getUserInfoAsync().then(function(userInfo) {
+            $scope.canEditRefresherExplorationId = (
```

Comment:
Should we declare this false at the outset to begin with, or is the intent to leave it undefined? (Just wanted to establish what the pattern is.)

Same question throughout, in other files.

Reply:
Define this scope variables to some default value.

Reply:
I don't see the default value...